packages/makeup-modal/src/index.js

```diff
@@ -3,12 +3,67 @@
 const keyboardTrap = require('makeup-keyboard-trap');
 const screenreaderTrap = require('makeup-screenreader-trap');
 
+let hoistEl;
+let hoistedElementPlaceholder;
+let containerDiv;
+let bodyChildIndexes = [];
+
+function unhoist() {
+    if (hoistEl) {
+        if (containerDiv) {
```

Comment:
This check for containerDiv is probably unneccesary, given that we have already checked for hoistEl.